fiftyone/core/annotation/generate_label_schemas.py

```
@@ -436,3 +518,54 @@ def _handle_str(collection, field_name, is_list, settings, scan_samples):
+
+    current_path = ""
+    for part in parts:
+        current_path = f"{current_path}.{part}" if current_path else part
+        pipeline.append({"$unwind": f"${current_path}"})
+
+    return pipeline
+
+
+def build_distinct_pipeline(
+    path: str,
+    sample_size: int = None,
```

Ruff (0.14.14)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)